src/admin-dhis2/components/AdminDhis2ServersPage/AdminDhis2ServersPage.jsx

```diff
@@ -19,34 +19,43 @@ import getService from '../../../react-components/utils/angular-utils';
 import Table from '../../../react-components/table/table';
 import TrashButton from '../../../react-components/buttons/trash-button';
 import ResponsiveButton from '../../../react-components/buttons/responsive-button';
+import confirmDialogAlert from '../../../react-components/modals/confirm';
 
 const AdminDhis2ServersPage = () => {
     const [serversParams, setServersParams] = useState([]);
 
-    const serversData = useMemo(
+    const serverServices = useMemo(
```

Comment:
I think we can rename this const to `serverService` without s at the end

Reply:
ok. I can rename this const.